internal/decoder/stream.go

```diff
@@ -281,6 +281,7 @@ func (s *Stream) skipObject(depth int64) error {
 						s.cursor = cursor
 						if s.read() {
 							_, cursor, p = s.statForRetry()
+							cursor++
```

Comment:
Please use `s.stat()` instead of `s.statForRetry()` and remove `cursor++`

Reply:
This looks clearer! I reflected your review in the following commit.